### PR TITLE
Create Models

### PR DESCRIPTION
**Pull request by gharper235**

### CODE REVIEW

File: models/FanPage.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const mongoose = require('mongoose');
+
+const userSchema = new mongoose.Schema(
+  {
+    author: [{
+      type: mongoose.Types.ObjectId,
+      ref: 'User'
+    }],
```
Comment (ifrah1) [inline models/FanPage.js]: I would change this to just an user object so you can get rid of the [ ]

File: models/FanPage.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+const mongoose = require('mongoose');
+
+const userSchema = new mongoose.Schema(
+  {
+    author: [{
+      type: mongoose.Types.ObjectId,
```
Comment (ifrah1) [inline models/FanPage.js]: not sure if it should be 'mongoose.Schema.Types.ObjectId'? double check that

File: models/FanPage.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    },
+    page_title: {
+      type: String,
+      required: true,
+    },
+    page_bio: {
+        type: String,
+        required: false,
+    },
+    user_tracks: {
```
Comment (ifrah1) [inline models/FanPage.js]: For all the variables we want to use camel case for our application ex: userTracks

In ERD we use snake case

File: models/FanPage.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    user_albums: {
+        type: String,
+        required: false,
+    },
+    user_shows: {
+        type: String,
+        required: false,
+    },
+    upvote: [{
+      type: mongoose.Types.ObjectId,
```
Comment (ifrah1) [inline models/FanPage.js]: not sure if it should be 'mongoose.Schema.Types.ObjectId'? double check that

**Review by ifrah1 — CHANGES_REQUESTED** [review]

File: models/FanPage.js (hunk 2 above)
Comment (gharper235) [reply models/FanPage.js]: According to Stack Overflow they are both the same thing. looks like they are interchangeable.... https://stackoverflow.com/questions/28997636/should-i-use-schema-types-objectid-or-schema-objectid-when-defining-a-mongoose-s

File: models/FanPage.js (hunk 4 above)
Comment (gharper235) [reply models/FanPage.js]: According to Stack Overflow they are both the same thing. looks like they are interchangeable.... https://stackoverflow.com/questions/28997636/should-i-use-schema-types-objectid-or-schema-objectid-when-defining-a-mongoose-s

**Pull request closed without merging**